src/main/java/org/folio/services/invoice/InvoiceCancelService.java

```diff
@@ -104,13 +111,30 @@ private void validateCancelInvoice(Invoice invoiceFromStorage) {
     List<Invoice.Status> cancellable = List.of(Invoice.Status.APPROVED, Invoice.Status.PAID);
     if (!cancellable.contains(invoiceFromStorage.getStatus())) {
       List<Parameter> parameters = Collections.singletonList(
-        new Parameter().withKey("invoiceId").withValue(invoiceFromStorage.getId()));
+        new Parameter().withKey(INVOICE_ID).withValue(invoiceFromStorage.getId()));
       Error error = CANNOT_CANCEL_INVOICE.toError()
         .withParameters(parameters);
       throw new HttpException(422, error);
     }
   }
 
+  /**
+   * Performs validation of budget statuses associated with an invoice.
+   *
+   * @param invoice The invoice.This parameter is necessary to extract the associated budgets.
+   * @param lines The list of invoice lines. This parameter is necessary to extract the associated budgets.
+   * @param requestContext The request context providing additional information.
+   * @return A `Future` of type `Void`, representing the result of the validation. If the future succeeds,
+   * it indicates that the validation has been successfully completed, and active budgets have been extracted
+   * @throws HttpException If no active budgets are found, an exception is thrown.
+   */
+  private Future<Void> validateBudgetsStatus(Invoice invoice, List<InvoiceLine> lines, RequestContext requestContext) {
```

Comment:
maybe validateBudgetsActive()?

Reply:
Updated javadoc, please take a look. I believe it will be worthy replacement.